platform/viewer/pluginConfig.json

```diff
@@ -1,48 +1,55 @@
 {
   "extensions": [
+    {
+      "packageName": "@ohif/extension-default"
+    },
     {
       "packageName": "@ohif/extension-cornerstone",
-      "version": "3.0.0"
+      "default": false
```

Comment:
why we are getting rid of the version here?

Reply:
It is meaningless - only external packages are meaningful to have the version number.

Reply:
maybe right now is meaningless but in future when we package properly and version them it is needed. So I don't think we remove and add back again, just let it be there